ts/src/bybit.ts

```diff
@@ -3465,18 +3465,15 @@ export default class bybit extends Exchange {
         this.checkRequiredSymbol ('createOrder', symbol);
         const market = this.market (symbol);
         symbol = market['symbol'];
-        const [ enableUnifiedMargin, enableUnifiedAccount ] = await this.isUnifiedEnabled ();
+        const values = await this.isUnifiedEnabled ();
+        const isUnifiedAccount = this.safeValue (values, 1);
         const isUSDCSettled = market['settle'] === 'USDC';
-        if (enableUnifiedAccount && !market['inverse']) {
+        if (isUnifiedAccount) {
             return await this.createUnifiedAccountOrder (symbol, type, side, amount, price, params);
-        } else if (market['spot']) {
-            return await this.createSpotOrder (symbol, type, side, amount, price, params);
-        } else if (enableUnifiedMargin && !market['inverse']) {
-            return await this.createUnifiedMarginOrder (symbol, type, side, amount, price, params);
         } else if (isUSDCSettled) {
             return await this.createUsdcOrder (symbol, type, side, amount, price, params);
         } else {
-            return await this.createContractV3Order (symbol, type, side, amount, price, params);
+            return await this.createNormalAccountOrder (symbol, type, side, amount, price, params);
         }
```

Comment:
@sc0Vu why are we keeping this? can't we just use v5?

Reply:
I'll remove this.